docs/finalproject/week-4/lesson.md

```diff
@@ -7,3 +7,5 @@ sidebar_label: Lesson
 See [Week 2](../week-2/lesson) for instructions for this week. 
 
 Make sure to check in again on the contributions per team members, as detailed on [Week 3](../week-3/lesson)
+
+The team should be asked to fill out the [handover document](https://docs.google.com/document/d/1hWrWS1kyHfJnzSeTOc7mwjElSJx_xtk6Down1Bzlaos/edit) once they have finished the final project development. This document is important to ensure the Product Owner and technical team that would continue working on this product has all necessary information about what has been done.
```

Comment:
```suggestion
Each team must fill out their [handover document](https://docs.google.com/document/d/1hWrWS1kyHfJnzSeTOc7mwjElSJx_xtk6Down1Bzlaos/edit) once they have finished the final project development. This document is important to ensure the Product Owner and technical team that would continue working on this product has all necessary information about what has been done.
```

Reply:
(Active language please)

Reply:
(Excellent point, thanks! It's the automatic translation in my head)
I have updated the handover document and tagged you.